src/Peachpie.Runtime/Utilities/DateTimeUtils.cs

```diff
@@ -34,5 +34,15 @@ public static double UtcToUnixTimeStampFloat(DateTime dt)
         {
             return (dt - UtcStartOfUnixEpoch).TotalSeconds;
         }
+
+        /// <summary>
+        /// Converts UNIX timestamp to <see cref="DateTime"/> representing UTC time.
+        /// </summary>
+        /// <param name="ts">Unix timestamp.</param>
+        /// <returns>Time.</returns>
+        public static DateTime UnixTimeStampToUtc(long ts)
+        {
+            return ts == 0 ? DateTime.MinValue : DateTimeOffset.FromUnixTimeSeconds(ts).UtcDateTime;
```

Comment:
it's not true, if `0` has a special meaning in cookies, please handle it in cookies, not in Runtime Utilities